src/itemAction/index.unit.tsx

```diff
@@ -54,26 +54,26 @@ describe('ItemAction', () => {
 
   describe('status', () => {
     it('Should not render Loader when status is not defined', () => {
-      const wrapper = shallow(<ItemAction>...</ItemAction>).dive()
+      const wrapper = mount(<ItemAction>...</ItemAction>)
```

Comment:
Why `mount` here ?

Reply:
We need to mount to access inside the `Item` from a parent component (before we used a `dive()` but since `Item` is wrapped by a hoc, we cannot do this anymore). I'm not usually a fan of diving into children, but for now it fixes the test.

Would be best to update the tests a bit once we migrate those components, to remove tests that are checking inside the children components, and thus testing the child behavior (which should already be done by `Item` itself).